alcs-frontend/src/app/services/authentication/authentication.service.ts

```diff
@@ -118,12 +118,14 @@ export class AuthenticationService {
       );
       return true;
     } catch (e) {
+      console.error(e);
```

Comment:
I assume this one is intentional here, correct?

Reply:
It is not, deleted!